easybuild/easyconfigs/s/spektral/spektral-1.1.0-foss-2021b-CUDA-11.4.1.eb

```
@@ -0,0 +1,38 @@
+# easybuild easyconfig
+#
+# John Dey <jfdey@fredhutch.org>
+#
+# Fred Hutchinson Cancer Center - Seattle Washington - US
+#
+easyblock = 'PythonPackage'
+
+name = 'spektral'
+version = '1.1.0'
+local_cuda_suffix = '-CUDA-%(cudaver)s'
+versionsuffix = local_cuda_suffix
```

I would just simplify this to:
```suggestion
versionsuffix = '-CUDA-%(cudaver)s'
```